proposed/container.md

```diff
@@ -97,9 +97,9 @@ into the same container (or another container) instead of the delegate container
 The interfaces and classes described as well as relevant exception are provided as part of the
 [psr/container](https://packagist.org/packages/psr/container) package. (still to-be-created)
 
-Packages providing a psr container implementation should declare that they provide psr/container-implementation 1.0.0
+Packages providing a psr container implementation should declare that they provide `psr/container-implementation` `1.0.0`.
```

Comment:
I'm not sure I agree with this formatting change; it feels unnecessary. Neither package names nor versions need to be formatted as code in any style guide I've ever used. Can you explain your justification for the change, please?